src/services/balances.rs

```diff
@@ -45,6 +45,7 @@ pub async fn balances(
             name: "Ether".to_string(),
             symbol: "ETH".to_string(),
             decimals: 18,
+            logo_url: "https://cryptologos.cc/logos/ethereum-eth-logo.png".to_string(),
```

Comment:
This was the default provided by Tobias. I'd prefer however if this would be a resource hosted by Gnosis in this case as this would be the default if the Chain Native Currency cannot be provided 🤔 

Another option would be to actually propagate the error if no native currency is provided since the spec specifies it as required. Defaulting to ETH might be the wrong assumption.

Reply:
I don't think this is my call, but my rational behind having a default value in the gateway is that in case we do have a balance object to return to the client but for some reason the native currency is not available, we would simply just show the wrong native coin info rather than discarding the entire response. 

It's similar to how we handle fiat conversions. If we have no conversion value, we simple set it to `0` instead of `panic`ing in the gw. Applying this rational to the native coin situation, seeing `ETH` in a network that doesn't correspond signals an error that we can (hopefully) easily address.

Reply:
I am very much with @jpalvarezl here. Either the `logo_url` is optional on both services or it is required on both services. We should not have any defaults (neither for the `name`, `symbol` or `decimals`) as everything should be provided by the config service.

Reply:
Also after looking at the code ... this `Err` case can never happen. We should unwrap or bail here.

Reply:
Default removed 63085c8